llvm/docs/DeveloperPolicy.rst

```diff
@@ -86,6 +86,11 @@ that notices of confidentiality or non-disclosure cannot be respected.
 Making and Submitting a Patch
 -----------------------------
 
+The LLVM community requires contributors to have a public email address
+associated with their GitHub commits, so please ensure that "Keep my email
+addresses private" is disabled in your
+`account settings <https://github.com/settings/emails>`_.
```

Comment:
I think it's worth elaborating on the motivation. Here's some suggested text:

The LLVM project uses email to communicate to contributors outside of the GitHub platform about their past contributions. Primarily, our buildbot infrastructure uses emails to contact contributors about build and test failures.